src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -137,8 +184,21 @@ const MOCK_API_RESPONSE_DELAY = 5000;
     chiColumnsReset: e => {
       console.log('chiColumnsReset', e);
     },
-    chiSelectedRowsChange: e => {
-      console.log('chiRowSelect', e);
+    chiSelectedRowsChange(data) {
+      this.$data.bulkActionConfig = {
+        ...this.$data.bulkActionConfig,
+        showBulkActions: true,
+        selectedRow: data,
```

Comment:
I think there is a mismatch here. When the Bulk actions component gets mounted, it goes up in the component hierarchy and gets registered in DataTable so that DataTable listens to its events and changes the rendered rows accordingly. This should be handled internally between BulkActions  and DataTable.